Python/compile.c

```diff
@@ -6910,14 +6910,14 @@ optimize_basic_block(basicblock *bb, PyObject *consts)
                             }
                         }
                         break;
-                    # if 0
+                    # if 1
```

Comment:
I get a scary error in test_sys_settrace.py:
```
PS C:\Users\gvanrossum\cpython> .\PCbuild\amd64\python.exe -m test test_sys_settrace
0:00:00 Run tests sequentially
0:00:00 [1/1] test_sys_settrace
Fatal Python error: PyFrame_BlockPop: block stack underflow
Python runtime state: initialized

Current thread 0x00005704 (most recent call first):
  File "C:\Users\gvanrossum\cpython\lib\test\test_sys_settrace.py", line 1550 in test_jump_over_return_in_try_finally_block
  File "C:\Users\gvanrossum\cpython\lib\test\test_sys_settrace.py", line 1179 in run_test
  File "C:\Users\gvanrossum\cpython\lib\test\test_sys_settrace.py", line 1207 in test
  File "C:\Users\gvanrossum\cpython\lib\unittest\case.py", line 549 in _callTestMethod
  File "C:\Users\gvanrossum\cpython\lib\unittest\case.py", line 592 in run
  File "C:\Users\gvanrossum\cpython\lib\unittest\case.py", line 652 in __call__
  File "C:\Users\gvanrossum\cpython\lib\unittest\suite.py", line 122 in run
  File "C:\Users\gvanrossum\cpython\lib\unittest\suite.py", line 84 in __call__
  File "C:\Users\gvanrossum\cpython\lib\unittest\suite.py", line 122 in run
  File "C:\Users\gvanrossum\cpython\lib\unittest\suite.py", line 84 in __call__
  File "C:\Users\gvanrossum\cpython\lib\unittest\suite.py", line 122 in run
  File "C:\Users\gvanrossum\cpython\lib\unittest\suite.py", line 84 in __call__
  File "C:\Users\gvanrossum\cpython\lib\test\support\testresult.py", line 169 in run
  File "C:\Users\gvanrossum\cpython\lib\test\support\__init__.py", line 959 in _run_suite
  File "C:\Users\gvanrossum\cpython\lib\test\support\__init__.py", line 1082 in run_unittest
  File "C:\Users\gvanrossum\cpython\lib\test\libregrtest\runtest.py", line 210 in _test_module
  File "C:\Users\gvanrossum\cpython\lib\test\libregrtest\runtest.py", line 246 in _runtest_inner2
  File "C:\Users\gvanrossum\cpython\lib\test\libregrtest\runtest.py", line 282 in _runtest_inner
  File "C:\Users\gvanrossum\cpython\lib\test\libregrtest\runtest.py", line 154 in _runtest
  File "C:\Users\gvanrossum\cpython\lib\test\libregrtest\runtest.py", line 194 in runtest
  File "C:\Users\gvanrossum\cpython\lib\test\libregrtest\main.py", line 423 in run_tests_sequential
  File "C:\Users\gvanrossum\cpython\lib\test\libregrtest\main.py", line 521 in run_tests
  File "C:\Users\gvanrossum\cpython\lib\test\libregrtest\main.py", line 694 in _main
  File "C:\Users\gvanrossum\cpython\lib\test\libregrtest\main.py", line 641 in main
  File "C:\Users\gvanrossum\cpython\lib\test\libregrtest\main.py", line 719 in main
  File "C:\Users\gvanrossum\cpython\lib\test\__main__.py", line 2 in <module>
  File "C:\Users\gvanrossum\cpython\lib\runpy.py", line 86 in _run_code
  File "C:\Users\gvanrossum\cpython\lib\runpy.py", line 196 in _run_module_as_main

Extension modules: _testcapi, _overlapped (total: 2)
```
@markshannon You changed that file last. Should I just give up on RETURN_CONST/RETURN_NONE for now?

Reply:
Note: _overlapped is listed as a 3rd party C extension, whereas it's part of the stdlib. I forgot this one and I created https://github.com/python/cpython/pull/25122 to add it to sys.stdlib_module_names :-)